lib/fontcustom/scripts/generate.py

```diff
@@ -140,6 +141,6 @@ def createGlyph( name, source, code ):
 
 finally:
     manifestfile.seek(0)
-    manifestfile.write(json.dumps(manifest, indent=2, sort_keys=True))
+    manifestfile.write(json.dumps(manifest, indent=2, sort_keys=True).encode('utf-8').decode('utf-8'))
```

Comment:
Hello Jake,

I should say it before it was my first time i did something in python and i was aware of this Unicode exceptions between two major versions. So i was hoping someone give me an hint to improve it, because i dont need to learn python to understand that making an encode().decode() is not the best practice...

About 'latin1, first i tried to encode with 'utf-8' but i didnt work out and then i found this post -> https://stackoverflow.com/questions/23917729/switching-to-python-3-causing-unicodedecodeerror , i tried latin1 and worked... Honestly, i dont understand why... i thought utf-8 was enough for any kind character.